scvi/module/_mixupvae.py

```diff
@@ -157,11 +168,17 @@ def __init__(
         extra_encoder_kwargs: Optional[dict] = None,
         extra_decoder_kwargs: Optional[dict] = None,
         # MixUpVAE specific arguments
+        n_pseudobulks: Tunable[int] = 1,
+        n_cells_per_pseudobulk: Tunable[Optional[int]] = None,
         signature_type: Tunable[str] = "pre_encoded",
         loss_computation: Tunable[str] = "latent_space",
         pseudo_bulk: Tunable[str] = "pre_encoded",
         mixup_penalty: Tunable[str] = "l2",
+        mixup_penalty_aggregation: Tunable[str] = "mean",
+        average_variables_mixup_penalty: Tunable[bool] = False,
     ):
```

Comment:
From these new hyperparameters, 

which ones will be included in the stability experiments ?

Reply:
I feel like the list of hyperparameters is starting to become hard to manage. After the stability experiments I suggest we clean up the class and remove hyperparameters that remain fixed / haw negligible effect from the constants.

Reply:
Not sure yet, I want to first understand the overall effect of having several pseudobulks in a same batch.
But for instance the `average_variables_mixup_penalty ` is set to False for the general use case, but I think would be better as True for the case where the MixUpLoss is computed in the reconstructed space, since otherwise it gets huge...
But agreed on the second comment

Reply:
I think we talked about this but the reconstruted loss by be huge because the decoder is not trying to fit the pseudobulk data (no backprop) - so we can try learning w/ the  pseudobulk reconstruction and see if the problem persists. I'll look into that after we merge this.

Reply:
Well it's also because by summing across the variable dimension gives a huge number if the variable dimension is big (eg in the input space). But yes to investigate